v3/src/components/axis/models/axis-model.test.ts

```diff
@@ -1,8 +1,9 @@
+import {getSnapshot, types} from "mobx-state-tree"
 import {
   AxisModel, AxisModelUnion, CategoricalAxisModel, EmptyAxisModel, IAxisModelUnion,
   isCategoricalAxisModel, isEmptyAxisModel, isNumericAxisModel, NumericAxisModel
 } from "./axis-model"
-import {getSnapshot, types} from "mobx-state-tree"
+import { AppHistoryService } from "../../../models/history/app-history-service"
```

Comment:
Could/should there be a `TestHistoryService` available in `src/test`? It seems weird for "low-level" jest tests to be reaching out for the application's history service.

Reply:
Yes they could be switched to use a Test or Noop history service. I was going to add that when I started moving tests over that are within to the "library". But it does seem good to add it now so I don't forget to update the tests that are outside of the "library".

Reply:
However on second thought, given the amount of files changed in this PR. I'd rather get it merged sooner than later. So I'm going to punt this change. I've added it to the the document where I'm tracking what is left to finish.